internal/view/login.html

```diff
@@ -149,10 +148,7 @@
 					});
 				},
 				loadSetting() {
-					var opts = JSON.parse(localStorage.getItem("shiori-account")) || {},
-						NightMode = (typeof opts.config.NightMode === "boolean") ? opts.configNightMode : false;
-
-					this.NightMode = NightMode;
+                    document.body.classList.remove("light", "dark");
```

Comment:
Does login page have any class set up? Since the user is not logged in it should just follow the system setting, right?